wear/app/build.gradle.kts

```diff
@@ -46,6 +46,14 @@ android {
             excludes += "/META-INF/{AL2.0,LGPL2.1}"
         }
     }
+
+    lint {
+        disable += "ExtraTranslation"
+        disable += "ObsoleteLintCustomCheck"
+        disable += "OldTargetAPI"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/2674cbe1cfb2698be3988559c9e73338e0feccee/wear/app/build.gradle.kts#L53"></span>Unknown issue id "OldTargetAPI". Did you mean 'OldTargetApi' (Target SDK attribute is not targeting latest version) ?</td>
    </tr>
  </tbody>
</table>